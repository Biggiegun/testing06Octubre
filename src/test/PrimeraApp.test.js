import React from 'react'
import {shallow, configure} from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import PrimeraApp from '../PrimeraApp'

configure({adapter: new Adapter()})
describe('Pruebas <PrimeraApp/>', ()=>{
    test('Validar render de <PrimeraApp/>', ()=>{
        const saludoC = 'Hola clase'
        const subTitulo='soy un subtitulo'
        const compo = shallow(<PrimeraApp saludo={saludoC} subtitulo={subTitulo}/>)

        const textParrafo = compo.find('p').text().trim();

        expect(textParrafo).toBe(subTitulo)
        expect(compo).toMatchSnapshot();
    })
})