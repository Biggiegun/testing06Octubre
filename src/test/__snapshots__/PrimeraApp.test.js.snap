// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas <PrimeraApp/> Validar render de <PrimeraApp/> 1`] = `ShallowWrapper {}`;
